frank-doc-doclet/src/test/java/org/frankframework/frankdoc/testtarget/examples/unsafe/Unsafe.java

```diff
@@ -0,0 +1,18 @@
+package org.frankframework.frankdoc.testtarget.examples.unsafe;
+
+public class Unsafe {
+
+	/**
+	 * Documentation for this unsafe attribute.
+	 */
+	@org.frankframework.doc.Unsafe
+	public void setUnsafeAttribute(String attribute) {
+	}
```

Comment:
```suggestion
	/**
	 * Documentation for this unsafe attribute.
	 */
	@org.frankframework.doc.Unsafe
	public void setUnsafeAttribute(String attribute) {
	}

	/**
	 * Documentation for this unsafe attribute without period at the end
	 */
	@org.frankframework.doc.Unsafe
	public void setUnsafeAttribute2(String attribute) {
	}
```

Zou jij kunnen zorgen dat hij als de voorgaande zin geen punt `.` heeft, die toe kunnen voegen?